LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDevice.cs

```diff
@@ -31,6 +31,8 @@ public class LoRaDevice : IDisposable, ILoRaDeviceRequestQueue
         /// </summary>
         internal const ushort DefaultJoinValues = 0;
 
+        internal static readonly TimeSpan TwinUpdateTimeout = TimeSpan.FromSeconds(10);
```

Comment:
let's move this to the LoRaDeviceClient instead and make it the default timeout. Seems odd, to reference the default timeout in the LoRaDeviceClient from here. The default should be driven by the device client wrapper and callers can adjust it if needed.

Reply:
Done in f8eb224